### ISSUE TITLE
Add flag to disable sharing qr code generator

### ISSUE THREAD

**Issue by networkException** [Eloston/ungoogled-chromium#1574]
**Is your feature request related to a problem? Please describe.**
In chromium versions prior to 92 there existed a flag to disable the then recently introduced qr code generator for sharing. Personally the addition of the corresponding context menu entry as well as omnibar icon does not add any value to the browser and I would prefer to have a choice to disable them.

**Describe the solution you'd like**
The addition of a flag to remove ui elements for the sharing qr code generator.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by Ahrotahn** [comment]
I can group this in with the PR for the tabsearch button removal patch once I get them set up.  
Until then the feature can be removed by applying this patch:
```diff
--- a/chrome/browser/ui/qrcode_generator/qrcode_generator_bubble_controller.cc
+++ b/chrome/browser/ui/qrcode_generator/qrcode_generator_bubble_controller.cc
@@ -22,10 +22,7 @@
 
 // static
 bool QRCodeGeneratorBubbleController::IsGeneratorAvailable(const GURL& url) {
-  if (!url.SchemeIsHTTPOrHTTPS())
     return false;
-
-  return true;
 }
 
 // static
--- a/chrome/browser/ui/views/location_bar/location_bar_view.cc
+++ b/chrome/browser/ui/views/location_bar/location_bar_view.cc
@@ -245,7 +245,6 @@
     // the left most icon.
     params.types_enabled.push_back(PageActionIconType::kSendTabToSelf);
     params.types_enabled.push_back(PageActionIconType::kClickToCall);
-    params.types_enabled.push_back(PageActionIconType::kQRCodeGenerator);
     if (base::FeatureList::IsEnabled(kSharedClipboardUI))
       params.types_enabled.push_back(PageActionIconType::kSharedClipboard);
     if (base::FeatureList::IsEnabled(kWebOTPCrossDevice))
```

**Comment by networkException** [comment]
amazing, thank you!

**Comment by BachoSeven** [comment]
I think this can be closed since #1591 was merged
